src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -61,14 +74,14 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     "maproom--unclickable": isUnclickable,
     "maproom--iframe": isMapFrame,
     "maproom--always-show-label":
-      shouldShowLabel &&
-      (venue.roomVisibility === RoomVisibility.nameCount ||
-        (venue.roomVisibility === RoomVisibility.count && hasRecentRoomUsers)),
+      shouldShowLabel && (roomLabelConditions || venueLabelConditions),
   });
 
   const titleClasses = classNames("maproom__title", {
     "maproom__title--count":
-      !isCovertRoom && venue.roomVisibility === RoomVisibility.count,
+      !isCovertRoom &&
+      (room.visibility === RoomVisibility.count ||
```

Comment:
Isn't the venue config supposed to be with priority, over the individual portal setting?

Reply:
@sofisparkle What is the right priority for portal visibility?
The `Venue` settings are more or less important than portal ones.

Reply:
based on Sofi's input - the venue config has priority over the portal settings

Reply:
> If a user sets Always in venue config, then the test portal should be set to always

> the individual portal settings should read "override global settings for this portal"

The portal settings have more priority to override the Venue config